.github/workflows/sync.yml

```diff
@@ -0,0 +1,194 @@
+name: Sync
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+env:
+  BOT_USERNAME: "exercism-bot"
+  GIT_USERNAME: "Exercism Bot"
+  GIT_EMAIL: "66069679+exercism-bot@users.noreply.github.com"
```

Comment:
How did you decide upon this email?

Reply:
Jeremy told me to use it :D  I think it's just the account's noreply address that github provides.

Eventually we may want to replace it with some kind of `botowner@exercism.io` address